README.md

```diff
@@ -1,43 +1,43 @@
-## CloudFoundry PHP Example Application: Drupal 
+## cloud.gov example application: Drupal 7
```

Comment:
@jcscottiii Looks like I wrote my reply to 'a question' while you were writing the other comments. :) +1 to creating two-level subdirectories once we have Drupal 8 ready to deploy.